Complete Italian translation

mpaglia0 opened the pull request.

aurelienpierre commented:
Thanks !

aurelienpierre commented:
Wait, you removed an SVG watermark (not sure why) I see no .po file. Are you sure you commited everything properly ?

mpaglia0 commented:
Ciao,
I removed hasselblad.svg since it is not a watermark but a simply frame (and is usable only on square photo) and added logo.svg
As regards it.po I think I missed something... I will check
